src/screens/FeedScreen/index.js

```diff
@@ -224,12 +223,6 @@ const FeedScreen = (props) => {
     />
   );
 
-  const onWillSendViewPostTime = (event) => {
-    if (isSamePostViewed(event)) return;
-    sendViewPostTime();
-    updateViewPostTime(event);
-  };
-
   const renderItem = ({item, index}) => {
     if (item.dummy) return <React.Fragment key={index} />;
     if (updateMoreText && updateIndex === index) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onWillSendViewPostTime` function has been removed from this location and moved to the `onMomentumScrollEnd` event handler. This seems to be part of the refactoring effort to improve the handling of post view times. Ensure that this change doesn't introduce any unexpected behavior.
```diff
-  const onWillSendViewPostTime = (event) => {
-    if (isSamePostViewed(event)) return;
-    sendViewPostTime();
-    updateViewPostTime(event);
-  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->